dockerfiles/docker-compose.yml

```diff
@@ -29,7 +29,8 @@ services:
     volumes:
       - ${PWD}/common/dockerfiles/entrypoints/common.sh:/usr/src/app/docker/common.sh
       - ${PWD}/common/dockerfiles/entrypoints/proxito.sh:/usr/src/app/docker/proxito.sh
-      - ${PWD}/../readthedocs-ext:/usr/src/app/checkouts/readthedocs-ext
+      - ${PWD}/${RTDDEV_PATH_EXT}:/usr/src/app/checkouts/readthedocs-ext
+      - ${PWD}/${RTDDEV_PATH_EXT_THEME}:/usr/src/app/checkouts/readthedocs-ext-theme
```

Comment:
You could also use defaults `${RTDDEV_PATH_EXT:-../readthedocs-ext}`

https://docs.docker.com/compose/compose-file/#variable-substitution

Reply:
Thanks! I was looking for something like this and wasn't finding it in the docs. That's a much better solution